.github/ISSUE_TEMPLATE/user_bug_report.yml

```diff
@@ -0,0 +1,89 @@
+name: User bug report
```

Comment:
Changing the name of the template file will break all the links we've made to it. Please keep the same name.

Reply:
Oops, I see we have to change the extension anyway.